collect_app/src/main/java/org/odk/collect/android/activities/FillBlankFormActivity.kt

```diff
@@ -218,11 +197,15 @@ class FillBlankFormActivity :
     private fun onMapButtonClick(id: Long) {
         permissionsProvider.requestLocationPermissions(
             this,
-            object : PermissionListener {
+            object :
+                PermissionListener {
                 override fun granted() {
-                    selectFormFromMap.launch(
-                        Input(currentProjectProvider.getCurrentProject().uuid, id)
-                    )
+                    val intent =
```

Comment:
This could be just:
```
Intent(this@FillBlankFormActivity, FormMapActivity::class.java).also {
                        it.putExtra(FormMapActivity.EXTRA_FORM_ID, id)
                        startActivity(it)
                    }
```

instead of:

```                   
 val intent =
                        Intent(this@FillBlankFormActivity, FormMapActivity::class.java).also {
                            it.putExtra(FormMapActivity.EXTRA_FORM_ID, id)
                        }

                    startActivity(intent)

```